src/js/components/Larivaar/util.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+const matrasThatAppearAtRightSideOfChar = "wIuUyYoOW"
+const halfCharThatAppearAtRightSideOfChar = "Í´R@˜®"
+const nasalSoundCaharacter = "NMµ"
+const sihari = "i";
+const allMatrasHalfCharAndNasalSoundChar = sihari + matrasThatAppearAtRightSideOfChar + halfCharThatAppearAtRightSideOfChar + nasalSoundCaharacter
+const unicodeMatras = "ਾਿੀੁੂੇੈੋੌੰਂ"
+
+export function fixLarivaar(word, unicode, larivaarAssistColor, indexVal) {
```

Comment:
Since we are returning JSX array, maybe we should make this a component. 

```jsx
<LarivaarWord unicode={unicode} larivaarAssist={larivaarAssist}>{val}</LarivaarWord>
```